Library/Formula/osxfuse.rb

```diff
@@ -25,6 +25,9 @@ class Osxfuse < Formula
 
   conflicts_with "fuse4x", :because => "both install `fuse.pc`"
 
+  # allow building with Xcode 6.3
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!